services/director-v2/src/simcore_service_director_v2/models/schemas/clusters.py

```diff
@@ -42,7 +42,7 @@ class WorkersDict(DictModel[AnyUrl, Worker]):
 
 class Scheduler(BaseModel):
     status: str = Field(..., description="The running status of the scheduler")
-    workers: WorkersDict
+    workers: Optional[WorkersDict] = Field(default_factory=dict)
```

Comment:
### before
 ``worker`` is required and is a dict
### now
 ``worker`` is **not** required, defaults to ``{}`` but is also nullable ``None``. So now you have two states that means the same i.e. ``workers=None`` and ``workers={}``.  Do we really need these two states:
- PROS: both ``bool(wokers) == False``
- CONS: 
    - ``workers`` it is NOT guaranteed to be iteratable, so we always need to check, e.g. this **might** fail
  ```python
   for worker in scheduler.workers:
    ...
   ```
   - json-schema auto generation does not account for nullable

### Suggestion: 
there is not need to make it nullable i.e. remove the ``Optional``

Reply:
ok but the point here is different since the dask'client sometimes returns an empty object.

Reply:
the i believe it indeed is justified :)